Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -57,19 +57,28 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
     /// **default**: false
     @objc open var keepPositionOnRotation: Bool = false
     
-    /// the object representing the left y-axis
-    internal var _leftAxis: YAxis!
+    /// The left y-axis object. In the horizontal bar-chart, this is the
+    /// top axis.
+    @objc open internal(set) var leftAxis: YAxis!
     
-    /// the object representing the right y-axis
-    internal var _rightAxis: YAxis!
+    /// The right y-axis object. In the horizontal bar-chart, this is the
+    /// bottom axis.
+    @objc open internal(set) var rightAxis: YAxis!
+
+    /// The left Y axis renderer. This is a read-write property so you can set your own custom renderer here.
+    /// **default**: An instance of YAxisRenderer
+    @objc open var leftYAxisRenderer: YAxisRenderer!
 
-    internal var _leftYAxisRenderer: YAxisRenderer!
-    internal var _rightYAxisRenderer: YAxisRenderer!
+    /// The right Y axis renderer. This is a read-write property so you can set your own custom renderer here.
+    /// **default**: An instance of YAxisRenderer
+    @objc open var rightYAxisRenderer: YAxisRenderer!
     
     internal var _leftAxisTransformer: Transformer!
```

Comment:
how about these two and some others with `_`?  Like `_tapGestureRecognizer` It means we still need `_` for some?

Reply:
It means I didn't touch them. I think they shouldn't have the `_`.